test/linters/params/swift-format.js

```diff
@@ -0,0 +1,83 @@
+const { join } = require("path");
+
+const swift_format = require("../../../src/linters/swift-format");
+
+const testName = "swift-format"
+const linter = swift_format;
+const extensions = ["swift"];
+
+function getLintParams(dir) {
+  const warning1 = `${join(
+    dir,
+    "file2.swift",
+  )}:2:22: warning: [DoNotUseSemicolons]: remove ';'`;
+  const warning2 = `${join(
+    dir,
+    "file1.swift",
+  )}:3:35: warning: [RemoveLine]: remove line break`;
+  const warning3 = `${join(
+    dir,
+    "file1.swift",
+  )}:7:1: warning: [Indentation] replace leading whitespace with 2 spaces`;
+  const warning4 = `${join(
+    dir,
+    "file1.swift",
+  )}:7:23: warning: [Spacing]: add 1 space`;
```

Comment:
Does `swift-format` only return messages with the `warning` level (no `error`s)? If so, we might want to add these to `lintResult.error` instead of `lintResult.warning`. I believe GitHub marks a check as passed if there are warnings but no errors. I've implemented it the same way [here](https://github.com/samuelmeuli/lint-action/blob/82ccadc7026bcdf40b193e8a599b7467b3e1c6e0/src/linters/swiftformat.js#L64-L65).

Reply:
Turns out that is the case, I'll update the change to ensure that it treats it as an error.